drivers/rtc/rtc-pcf2127.c

```diff
@@ -568,6 +568,7 @@ static int pcf2127_probe(struct device *dev, struct regmap *regmap,
 			 int alarm_irq, const char *name, bool is_pcf2127)
 {
 	struct pcf2127 *pcf2127;
+	bool ts_off = false;
```

Comment:
Now the timestamping is not disabled, right?

Reply:
your right, will fix that